Update README.md for release 4.22.1.0 (#377)
heremaps/here-sdk-examples: README.md

- **HelloMapCarPlay**: Shows how CarPlay can be integrated into the _HelloMap_ app to display a map on an in-car head unit display (iOS only). Exclusively available for the _Explore Edition_ and the _Navigate Edition_.
- **HelloMapSurface**: Shows how to use the MapSurface class to render low-level graphic elements with OpenGL ES on top of the map. (Android only). Exclusively available for the _Explore Edition_ and the _Navigate Edition_.
- **Camera**: Shows how to change the target and the target anchor point and how to move to another location using custom map animations.

> - **MapFeatures**: Shows how to enable different map view features. Exclusively available for the _Navigate Edition_ (Android and Flutter only).

Reviewer: ```suggestion
- **MapFeatures**: Shows how to enable different map view features. Exclusively available for the _Navigate Edition_.
```

- **PositioningWithBackgroundUpdates**: Shows how to integrate HERE Positioning with background location updates on Android using a foreground service. Exclusively available for the _Navigate Edition_.
- **HikingDiary**: Shows how to record GPX traces with HERE Positioning. Exclusively available for the _Navigate Edition_.
- **Traffic**: Shows how to search for real-time traffic and how to visualize it on the map.
- **TruckGuidance**: Shows how the HERE SDK can be used to calculate routes specific for trucks. In addition, it shows many more truck-related features. Exclusively available for the _Navigate Edition_.
- **StandAloneEngine**: Shows how to use an engine without a map view.
- **IndoorMap**: Shows how to integrate private venues. Exclusively available for the _Navigate Edition_.
- **UnitTesting**: Shows how to mock HERE SDK classes when writing unit tests (the example app is available for the _Explore Edition_ and the _Navigate Edition_).

> - **GesturesKotlin**: Shows how to handle gestures (the example app is available for the _Explore Edition_ and the _Navigate Edition_).

Reviewer: ```suggestion
- **GesturesKotlin**: Shows how to handle gestures on the map view. Exclusively available for the _Explore Edition_ and the _Navigate Edition_.
```

> - **TrafficKotlin**: Shows how to search for real-time traffic and how to visualize it on the map (the example app is available for the _Explore Edition_ and the _Navigate Edition_).

Reviewer: ```suggestion
- **TrafficKotlin**: Shows how to search for real-time traffic and how to visualize it on the map. Exclusively available for the _Explore Edition_ and the _Navigate Edition_.
```

> - **CameraKotlin**: Shows how to change the target and the target anchor point and how to move to another location using custom map animations (the example app is available for the _Explore Edition_ and the _Navigate Edition_).

Reviewer: ```suggestion
- **CameraKotlin**: Shows how to change the target and the target anchor point and how to move to another location using custom map animations. Exclusively available for the _Explore Edition_ and the _Navigate Edition_.
```

> - **MapItemsKotlin**: Shows how to add circles, polygons and polylines, native views, 2D and 3D map markers to locate POIs (and more) on the map. 3D map markers are exclusively (the example app is available for the _Explore Edition_ and the _Navigate Edition_).

Reviewer: ```suggestion
- **MapItemsKotlin**: Shows how to add circles, polygons and polylines, native views, 2D and 3D map markers to locate POIs (and more) on the map. 3D map markers are exclusively. Exclusively available for the _Explore Edition_ and the _Navigate Edition_.
```

> - **SearchKotlin**: Shows how to search POIs and add them to the map. Shows also geocoding and reverse geocoding (the example app is available for the _Explore Edition_ and the _Navigate Edition_).

Reviewer: ```suggestion
- **SearchKotlin**: Shows how to search POIs and add them to the map. Shows also geocoding and reverse geocoding. Exclusively available for the _Explore Edition_ and the _Navigate Edition_.
```

> - **NavigationHeadless**: Shows how the HERE SDK can be set up to navigate without following a route in the most simplest way - and without showing a map view (the example app is available for the _Navigate Edition_).

Reviewer: ```suggestion
- **NavigationHeadless**: Shows how the HERE SDK can be set up to navigate without following a route in the most simplest way - and without showing a map view. Exclusively available for the _Navigate Edition_.
```

> - **RoutingWithAvoidanceOptions**: Shows how to calculate routes with AvoidanceOptions and how to use the SegmentDataLoader to retrieve data from the map. It also allows to pick segments from the map to avoid certain areas or roads (the example app is available for the _Navigate Edition_).

Reviewer: ```suggestion
- **RoutingWithAvoidanceOptions**: Shows how to calculate routes with AvoidanceOptions and how to use the SegmentDataLoader to retrieve data from the map. It also allows to pick segments from the map to avoid certain areas or roads. Exclusively available for the _Navigate Edition_.
```
